lib/features/parkings_view/repository/parkings_repository.dart

```diff
@@ -13,7 +13,7 @@ extension RefIntervalRefreshX on Ref {
   void setRefresh(Duration interval) {
     final timer = Timer.periodic(
       interval,
-      (t) => invalidateSelf(),
+      (t) => invalidate(parkingsRepositoryProvider),
```

Comment:
Unfortunetly this throws :( 
```console
[ERROR:flutter/runtime/dart_vm_initializer.cc(41)] Unhandled Exception: 'package:riverpod/src/framework/element.dart': Failed assertion: line 614 pos 11: 'listenable._origin != origin': A provider cannot depend on itself
```
basically `invalidateSelf()` and `invalidate(someProvider)` works the same with the difference the second one seems to throw when use inside the same provider (I didn't knew that actually, but make sense)